tests/commands/test_patch.py

```diff
@@ -0,0 +1,63 @@
+"""
+:Description: Tests the `patch` CLI
+"""
+
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from conda_recipe_manager.commands.patch import patch
+from tests.file_loading import TEST_FILES_PATH
+from tests.smoke_testing import assert_cli_usage
+
+
+def test_usage() -> None:
+    """
+    Smoke test that ensures rendering of the help menu
+    """
+    assert_cli_usage(patch)
+
+
+def test_faulty_recipe_file(fs: pytest.Function) -> None:
```

Comment:
```suggestion
def test_patch_cli_faulty_recipe_file(fs: pytest.Function) -> None:
```

This is more in-line with the naming scheme of the rest of the project. Refer to my other comment for more details. There's nothing necessarily wrong with the test name, but I want to try to be consistent. It also makes it easier to find similar tests if they share the same prefix.